e3sm_to_cmip/cmor_handlers/cLitter.py

```diff
@@ -17,6 +20,8 @@ def write_data(varid, data, timeval, timebnds, index, **kwargs):
     cLitter = (TOTLITC + CWDC)/1000.0
     """
     outdata = (data['TOTLITC'][index, :] + data['CWDC'][index, :])/1000.0
```

Comment:
To get around not a numpy array problem, I think use the values attribut to give a numpy ndarray.
outdata = (data['TOTLITC'][index, :].values + data['CWDC'][index, :].values)/1000.0

Reply:
Thanks, I have added `.values` for this handler.